src/source.js

```diff
@@ -0,0 +1,55 @@
+// import '@fortawesome/fontawesome-free/js/fontawesome';
+// import '@fortawesome/fontawesome-free/js/solid';
```

Comment:
- [ ] Comments: It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png)

Reply:
Thanks did notice I left them in